common/changes/@microsoft/api-documenter/api-readonly-mixin_2022-05-19-20-18.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-documenter",
+      "comment": "Adding readonlyMixin for Api Extractor",
```

Comment:
I think this is minor instead of a patch because it's adding new functionality, not patching a bug fix. WDYT? Also, consider updating the change comments to be more specific/meaningful per package. For example, for api-documenter: "Add support for rendering readonly information next to API items".

Reply:
Definitely agree it would be minor, I tend to view functionality as a new pathway of using existing code, but should be more liberal with that definition. Let me know if changed verbiage should be reworded.